app/src/main/java/org/mozilla/focus/activity/RightsActivity.java

```diff
@@ -1,6 +1,34 @@
 package org.mozilla.focus.activity;
 
-import android.app.Activity;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.View;
+import android.webkit.WebView;
 
-public class RightsActivity extends Activity {
+import org.mozilla.focus.R;
+
+public class RightsActivity extends AppCompatActivity {
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        setContentView(R.layout.activity_rights);
+        WebView mWebView = null;
+        mWebView = (WebView) findViewById(R.id.webview);
```

Comment:
I've been thinking about this a little - it would be good to share the same activity code for all of about/help/your-rights, and also some SUMO pages ("Set default browser"/"Learn more" for telemetry). Using the WebViewProvider has a potential bug/feature where image blocking being enabled makes SUMO pages a bit uglier - I filed #182 to make it easier to discuss that.